jsonschema-generator/src/main/java/com/github/victools/jsonschema/generator/SchemaGeneratorConfigBuilder.java

```diff
@@ -41,7 +42,9 @@ public class SchemaGeneratorConfigBuilder {
      * @return default ObjectMapper instance
      */
     private static ObjectMapper createDefaultObjectMapper() {
-        ObjectMapper mapper = new ObjectMapper();
+        ObjectMapper mapper = new ObjectMapper()
+                // enables pretty-printing by default (users can override this by supplying their own mapper)
```

Comment:
**suggestion**: Let's include the version number for easier future reference, as for the other settings below.
```suggestion
                // since version 4.32.0; pretty print by default (can be overridden by supplying explicit mapper)
```